From 7f60b2af6b829316e9015381c5a7b9f21898d71d Mon Sep 17 00:00:00 2001
From: Bin Han <bin.b.han@intel.com>
Date: Tue, 6 Jan 2015 10:01:15 +0800
Subject: [PATCH 246/255] atomisp2: two_pixels_per_clock is deprecated

two_pixels_per_clock is going to be deprecated by FW, driver
should use pixels_per_clock instead.

This is to fix #H1503902590.

Change-Id: I60266fe91a83fb742569dbcde91172adc67df4c0
Fixes-Issue: #H1503902590
Signed-off-by: Bin Han <bin.b.han@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/34981
Reviewed-by: Jing Tao <jing.tao@intel.com>
Approver: Wei Tao <wei.tao@intel.com>
Reviewed-by: Wei Tao <wei.tao@intel.com>
---
 .../media/pci/atomisp2/atomisp_driver/atomisp_compat_css20.c      | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_compat_css20.c b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_compat_css20.c
index 868d486..6d16c78 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_compat_css20.c
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_compat_css20.c
@@ -493,8 +493,8 @@ static void __dump_stream_config(struct atomisp_sub_device *asd,
 	dev_dbg(isp->dev, "stream_config.input_config.bayer_order=%d.\n",
 		s_config->input_config.bayer_order);
 
-	dev_dbg(isp->dev, "stream_config.2ppc=%d.\n",
-			s_config->two_pixels_per_clock);
+	dev_dbg(isp->dev, "stream_config.pixels_per_clock=%d.\n",
+			s_config->pixels_per_clock);
 	dev_dbg(isp->dev, "stream_config.online=%d.\n", s_config->online);
 	dev_dbg(isp->dev, "stream_config.continuous=%d.\n",
 			s_config->continuous);
@@ -2054,11 +2054,11 @@ void atomisp_css_input_set_two_pixels_per_clock(
 	int i;
 
 	if (asd->stream_env[ATOMISP_INPUT_STREAM_GENERAL]
-		.stream_config.two_pixels_per_clock == !!two_ppc)
+		.stream_config.pixels_per_clock == (two_ppc ? 2 : 1))
 		return;
 
 	asd->stream_env[ATOMISP_INPUT_STREAM_GENERAL]
-		.stream_config.two_pixels_per_clock = !!two_ppc;
+		.stream_config.pixels_per_clock = (two_ppc ? 2 : 1);
 	for (i = 0; i < IA_CSS_PIPE_ID_NUM; i++)
 		asd->stream_env[ATOMISP_INPUT_STREAM_GENERAL]
 		.update_pipe[i] = true;
-- 
1.9.1

